### PR TITLE
changes for adding volume collection with pool information

### PR DESCRIPTION
**Pull request by AnuBose**
For available resources api, add pool information for volume collections

### CODE REVIEW

File: v1/api/swagger/components/schemas/VolumeCollection.yaml [hunk 1]
```diff
@@ -22,3 +22,12 @@ properties:
   Description:
     type: string
     description: The description of the volume collection resource
+  PoolIDs:
```
Comment (patricia-dibenedetto) [inline v1/api/swagger/components/schemas/VolumeCollection.yaml:25]: StoragePoolIDs

File: v1/api/swagger/components/schemas/VolumeCollection.yaml [hunk 2]
```diff
@@ -22,3 +22,12 @@ properties:
...
+    description: Pool IDs that are valid for this volume collection
```
Comment (patricia-dibenedetto) [inline v1/api/swagger/components/schemas/VolumeCollection.yaml:26]: IDs of the storage pools that are valid for this volume collection as listed in the Storage Pools array

File: v1/api/swagger/components/schemas/StorageInventory.yaml [hunk 3]
```diff
@@ -26,3 +26,12 @@ properties:
     type: string
     format: uuid
     description: Unique ID of the storage pool as listed in the StoragePools array
+  VolumeCollectionIDs:
+    description: IDs of the volume collections for this flavour as listed in the Volume collections array
```
Comment (mchuang3) [inline v1/api/swagger/components/schemas/StorageInventory.yaml:30]:
```suggestion
    description: IDs of the volume collections for this flavor as listed in the Volume collections array
```

File: v1/api/swagger/components/schemas/VolumeCollection.yaml [hunk 4]
```diff
@@ -22,8 +22,8 @@ properties:
   Description:
     type: string
     description: The description of the volume collection resource
-  PoolIDs:
-    description: Pool IDs that are valid for this volume collection
+  StoragePoolIDs: 
+    description: IDs of the valid storage pools that are valid for this volume collection as listed in the Storage Pools array
```
Comment (patricia-dibenedetto) [inline v1/api/swagger/components/schemas/VolumeCollection.yaml:26]: remove first 'valid' word (dup)

**Review by patricia-dibenedetto — APPROVED** [review]

**Review by mchuang3 — APPROVED** [review]

**Pull request merged**